cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/DeployProgress/DeployProgress.tsx

```diff
@@ -160,65 +157,105 @@ function DeployProgress() {
                     </nav>
                 </div>
             </div>
-            {renderNextSteps()}
+            {displayNextSteps()}
         </>
     );
 
+    // Displays page title; style differs depending on creation of management or unmanaged cluster type
+    function displayPageTitle() {
+        if (clusterType && clusterType === DeploymentTypes.UNMANAGED_CLUSTER) {
```

Comment:
nit: you shouldn't need `clusterType &&` here, because if clusterType is null, the `===` will fail

Reply:
you're right, thanks